SwiftSnake/SwiftSnake/SKNodes/NodeConstants.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  NodeConstants.swift
+//  SwiftSnake
+//
+//  Created by Ellen Shapiro (Vokal) on 9/24/15.
+//  Copyright © 2015 Vokal. All rights reserved.
+//
+
+import Foundation
+
+enum NodeType: String {
+    case Food = "Food",
+    Background = "Background",
+    Space = "Space",
+    SnakeHead = "SnakeHead",
+    SnakeBody = "SnakeBody"
```

Comment:
in swift 2, you don't need to explicitly declare the enum string values (unless you want them to be different from the names)

also, add a trailing comma?